libraries/api-extractor-model/package.json

```diff
@@ -26,6 +26,7 @@
     "@rushstack/heft": "0.68.12",
     "@rushstack/heft-node-rig": "2.6.48",
     "@types/heft-jest": "1.0.1",
-    "@types/node": "20.17.19"
+    "@types/node": "20.17.19",
+    "typescript": "~5.7.3"
```

Comment:
Is the added dependency necessary? There doesn't appear to be any code changes to the package itself.

Reply:
This appears to be true for a few packages in this change

Reply:
Yes. These have decoupled local dependencies. When heft-node-rig is published, we can pull that in and remove these.